### ISSUE TITLE
Add check: not allow multiple PRs against different repos.

### ISSUE THREAD

**Issue by baranowb** [jboss-set/bug-clerk#107]
https://issues.jboss.org/browse/JBEAP-10449

**Comment by rpelisse** [comment]
Sounds good and rather easy to implement, but are *sure* there is no legitimate cases for one issue/several PRs? I'm just afraid fo False+ annoying people...

**Comment by baranowb** [comment]
not against different repositories/components.

**Comment by rpelisse** [comment]
Yes, you can probably easily check that all the PRs are against the same "component" by checking the name of the github project/organisation. Cool :) - let me know if you need help implementing it!
